README.md

```diff
@@ -33,3 +33,83 @@ Your pull requests are welcome but please keep them manageable and focussed.
 Equally your input on any pull requests we have in flight at any given time is invaluable to us, so please get involved.
 
 Thanks! :grin:
+
+
+## Running the example
+
+* Clone the repo
+* cd to `example` folder
+* run `flutter packages get` to install dependencies
+* `flutter run` will start the application on connected android / iOS device
+
+
+## Using the Realtime API
+
+##### Import the package
+```dart
```

Comment:
As you have lower down in this commit and aligning with the format of the file already, can you leave a line of vertical white space around headings please. While it may not change how it renders, it does affect readability of the source.

Reply:
Done@https://github.com/ably/ably-flutter/commit/083a0a518b96c8e773f98c8353af420b698beaa3